crates/oxc_linter/src/rules/jsx_a11y/aria_props.rs

```diff
@@ -0,0 +1,85 @@
+use oxc_ast::{
+    ast::{JSXAttributeItem, JSXAttributeName},
+    AstKind,
+};
+use oxc_diagnostics::{
+    miette::{self, Diagnostic},
+    thiserror::Error,
+};
+use oxc_macros::declare_oxc_lint;
+use oxc_span::Span;
+
+use crate::{context::LintContext, globals::VALID_ARIA_PROPS, rule::Rule, AstNode};
+
+#[derive(Debug, Error, Diagnostic)]
+#[error("eslint-plugin-jsx-a11y(aria-props):")]
+#[diagnostic(severity(warning), help("{1}"))]
```

Comment:
can we include an error message?

maybe
```
#[error("eslint-plugin-jsx-a11y(aria-props): Invalid aria prop.")]
#[diagnostic(severity(warning), help("`{1}` is an invalid aria attribute."))]

Reply:
@camc314 
Thanks! Included in https://github.com/oxc-project/oxc/pull/1797/commits/2d76503da2c49dd4a73199648d48f60a34a4874f 🙏

Reply:
🙏 thank you!